TestsExample/src/Test1096.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+
+import { Button, StyleSheet, View, Modal } from 'react-native';
+import {OverlayView} from 'react-native-screens';
+import {
+  createNativeStackNavigator,
+  NativeStackNavigationProp,
+} from 'react-native-screens/native-stack';
+import { NavigationContainer, ParamListBase } from '@react-navigation/native';
+
+function Home({
+  navigation,
+}: {
+  navigation: NativeStackNavigationProp<ParamListBase>;
+}) {
+  const [shown, setShown] = React.useState(true);
+  const [draggable, setDraggable] = React.useState(true);
+  const [isShowModal, setIsShowModal] = React.useState(false);
+  const [interceptTouches, setInterceptTouches] = React.useState(true);
+
+  return (
+    <View style={styles.container}>
+      <Button title="Show/hide window view" onPress={() => setShown(!shown)} />
+      <Button
+        title="Enable/disable dragging behavior"
+        onPress={() => setDraggable(!draggable)}
+      />
+      <Button
+        title="Enable/disable intercepting touches"
+        onPress={() => setInterceptTouches(!interceptTouches)}
+      />
+      <Button title="show rn modal" onPress={() => setIsShowModal(true)} />
+      <Button
+        title="Go to RNScreens modal"
+        onPress={() => navigation.navigate('Modal')}
+      />
+      <Modal
+        animationType="slide"
+        visible={isShowModal}
+        presentationStyle="pageSheet"
+      >
+        <View style={styles.container}>
+          <Button
+            title="dismiss rn modal"
+            onPress={() => setIsShowModal(false)}
+          />
+          <Button
+            title="Show/hide window view"
+            onPress={() => setShown(!shown)}
+          />
+        </View>
+      </Modal>
+      <OverlayView style={{position: 'absolute', width: '100%', height: '100%', justifyContent: 'center' }} shown={shown} draggable={draggable} interceptTouches={interceptTouches}>
```

Comment:
What happens if there are more than one of such components rendered in the tree?

Reply:
They will be rendered on top of each other in the same order as they were added to `React` view hierarchy at initialization since this code will be called: https://github.com/software-mansion/react-native-screens/pull/1096/files#diff-4882eafb07e471e6bd8bf6558dca65eacab42001de16e1dc1ca7b23c54ab765fR108 and then the `RNWindow` will move all containers to front in order they were added: https://github.com/software-mansion/react-native-screens/pull/1096/files#diff-174f03b4ee9b6d6719c80e57d200b2057b704709f1b3f83218e31b54819d22d4R11